src/Commands/InitCommand.php

```diff
@@ -273,7 +291,7 @@ protected function fillContacts(): void
 
         foreach (self::CONTACTS_ITEMS as $key => $title) {
             if ($link = $this->ask("Please enter a {$title}'s email", '')) {
-                if (!empty($link) && $key === self::CONTACTS_TEAM_LEAD) {
+                if (!empty($link) && $key === $this->emailTeamLead) {
```

Comment:
this condition would never return true as the key (you've only one key `manager`) would never be equals the email of the team lead, stored in this property